bot/code_review_bot/config.py

```diff
@@ -16,7 +16,8 @@
 import yaml
 
 PROJECT_NAME = 'code-review-bot'
-REPO_TRY = b'https://hg.mozilla.org/try'
+REPO_GECKO_TRY = 'https://hg.mozilla.org/try'
+REPO_NSS_TRY = 'https://hg.mozilla.org/projects/nss-try'
```

Comment:
We need to find a way to remove these when we want to make things more self-service

Reply:
All of that will disappear if we add our task directly into the taskgroup, instead of relying on pulse trigger.
If we can create the review bot's task directly, we can configure its payload to have the build target directly + all relevant infos: no need for finding what task does what

Reply:
:+1: